apps/admin-gui/src/app/shared/components/dialogs/edit-application-form-item-dialog/edit-application-form-item-dialog.component.html

```diff
@@ -10,163 +10,169 @@ <h1 mat-dialog-title>
     <div class="dialog-container" mat-dialog-content>
       <mat-tab-group color="primary" mat-stretch-tabs="false">
         <mat-tab label="{{'DIALOGS.APPLICATION_FORM_EDIT_ITEM.BASIC_SETTINGS' | translate}}">
-          <app-edit-application-form-item-line
-            [description]="'DIALOGS.APPLICATION_FORM_EDIT_ITEM.SHORT_NAME_DESCRIPTION' | translate"
-            [label]="'DIALOGS.APPLICATION_FORM_EDIT_ITEM.SHORT_NAME' | translate">
-            <mat-form-field class="w-100">
-              <input [(ngModel)]="applicationFormItem.shortname" matInput />
-            </mat-form-field>
-          </app-edit-application-form-item-line>
-          <app-edit-application-form-item-line
-            [description]="'DIALOGS.APPLICATION_FORM_EDIT_ITEM.INPUT_WIDGET_DESCRIPTION' | translate"
-            [label]="'DIALOGS.APPLICATION_FORM_EDIT_ITEM.INPUT_WIDGET' | translate">
-            {{applicationFormItem.type | applicationFormItemType}}
-          </app-edit-application-form-item-line>
-          <app-edit-application-form-item-line
-            [description]="'DIALOGS.APPLICATION_FORM_EDIT_ITEM.DISPLAY_ON_APPLICATION_DESCRIPTION' | translate"
-            [label]="'DIALOGS.APPLICATION_FORM_EDIT_ITEM.DISPLAY_ON_APPLICATION' | translate">
-            <section>
+          <mat-form-field subscriptSizing="dynamic" class="w-100">
+            <mat-label>{{'DIALOGS.APPLICATION_FORM_EDIT_ITEM.SHORT_NAME' | translate}}</mat-label>
+            <input [(ngModel)]="applicationFormItem.shortname" matInput readonly />
```

Comment:
Shortname should be editable.
```suggestion
            <input [(ngModel)]="applicationFormItem.shortname" matInput />
```

Reply:
Was testing how it looks for the item type and forgot to change it back, nice catch